paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -73,6 +73,14 @@ def parse_args():
         default=None,
         dest="source_id",
     )
+    parser.add_argument(
+        "--cluster-blacklist",
+        help="Space-separated list of clusters to not rightsize",
+        required=False,
+        nargs="+",
```

Comment:
Does the script work if you don't pass --cluster-blacklist? Just reading the docs on + vs * and want to double-check.